### PR TITLE
Packaging fixes

### PR DESCRIPTION
**Pull request by bear454**
With the need config structure, a few changes need to be made for packaging without a config.

### CODE REVIEW

**Review by jesusbv — APPROVED** [review]
LGTM

File: config/application.rb [hunk 1]
```diff
@@ -28,20 +28,30 @@ class Application < Rails::Application
...
+      ## The order determines the menu order - engines with no UI should be last
+      config.engines = config_for(:config)[:engines]
+
+      # Rancher source - for _helm_
+      config.x.rancher = OpenStruct.new(config_for(:config)[:rancher])
+      # kubernetes version
+      config.x.rancher_on_eks = OpenStruct.new(config_for(:config)[:rancher_on_eks])
+    rescue Exception
+      # don't crash if a config isn't present (necessary for packaging)
+      puts("WARNING: config.yml not present - the application will not be usable without it.")
```
Comment (jesusbv) [inline config/application.rb:49]: if the application is not usable, should this not be an error ?
and should the application stops after informing ?

Comment (bear454) [reply config/application.rb:49]: We use the Rails stack to create an empty database for the package, and to bundle the assets. Neither is possible if the config is required.

Comment (bear454) [reply config/application.rb:49]: I added a config as requirement in the specfile: 23e2a29 .

**Pull request merged**